demisto_sdk/commands/validate/validators/BA_validators/BA127_is_valid_context_path_depth_modified.py

```diff
@@ -0,0 +1,73 @@
+from demisto_sdk.commands.content_graph.objects.script import Script
+from demisto_sdk.commands.validate.validators.BA_validators.BA127_is_valid_context_path_depth import (
+    IsValidContextPathDepthValidator,
+)
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Union[Integration, Script]
+
+
+class IsValidContextPathDepthModifiedValidatorModified(
+    IsValidContextPathDepthValidator, BaseValidator[ContentTypes]
+):
+    expected_git_statuses = [GitStatuses.MODIFIED, GitStatuses.RENAMED]
+
+    def obtain_invalid_content_items(
+        self, content_items: Iterable[ContentTypes]
+    ) -> List[ValidationResult]:
+        results: List[ValidationResult] = []
+        for content_item in content_items:
+            if content_item.support != XSOAR_SUPPORT:
+                continue
+            old_content_item = content_item.old_base_content_object
+            if isinstance(content_item, Script):
+                old_script_paths = self.create_outputs_set(old_content_item)
+                new_script_paths = self.create_outputs_set(content_item)
+                changed_paths = set(new_script_paths).difference(old_script_paths)
+                invalid_paths = self.is_context_depth_larger_than_five(changed_paths)
+                if invalid_paths:
+                    results.append(
+                        ValidationResult(
+                            validator=self,
+                            message=self.error_message.format(
+                                "script", content_item.name, invalid_paths
+                            ),
+                            content_object=content_item,
+                        )
+                    )
+            else:
+                changed_paths_dict: Dict[str, Set[str]] = {}
+                old_command_paths = self.create_command_outputs_dict(old_content_item)
+                command_paths = self.create_command_outputs_dict(content_item)
+                for command_name, command_outputs in command_paths.items():
+                    if old_command_outputs := old_command_paths.get(command_name):
+                        changed_paths_dict[command_name] = command_outputs.difference(
+                            old_command_outputs
+                        )
+                invalid_paths = (
+                    self.is_context_depth_larger_than_five_integration_commands(
+                        changed_paths_dict
+                    )
+                )
+                if invalid_paths:
+                    results.append(
+                        ValidationResult(
+                            validator=self,
+                            message=self.error_message.format(
+                                "command",
+                                invalid_paths["command"],
+                                invalid_paths["wrong_paths"],
+                            ),
```

Comment:
correct me if wrong, but in this format, we print the list all commands with issues and then the list of all wrong paths.
I'd expect to have a better mapping, like:
in command 'a', the following contenxtPath depth are larger than five: contextPath_1, contextPath_2.
in command 'b', the following contenxtPath depth are larger than five: contextPath_1.

I'd suggest formatting the logical part of the error message before calling ValidationResult and then just format the string into the error message.